app/ui/window.js

```diff
@@ -216,7 +216,8 @@ module.exports = class Window {
       window.on(ev, () => rpc.emit('windowGeometry change'));
     }
     rpc.win.on('move', () => {
-      rpc.emit('move');
```

Comment:
Do you mind replacing `rpc.win` with `window`? I know you didn't write that part but It's a bit confusing the way it's currently written

Reply:
Yeah sure!